editoast/src/views/infra/railjson.rs

```diff
@@ -117,14 +115,22 @@ async fn get_railjson(
         neutral_sections = res[ObjectType::NeutralSection]
     );
 
-    Ok(HttpResponse::Ok()
-        .content_type(ContentType::json())
-        .append_header(("x-infra-version", infra_meta.version))
-        .body(railjson))
+    Ok((
+        StatusCode::OK,
+        [
+            (
+                header::CONTENT_TYPE.as_str(),
+                headers::ContentType::json().to_string(),
+            ),
```

Comment:
I believe you don't need this. If you wrap the variable `railjson` into a `Json(railjson)`, `axum` will automatically add the `Content-Type` header in the response.

Reply:
Wouldn't it also serialize it? `railjson` is a String containing JSON, we want to pass it verbatim.

Reply:
I don't see how setting up or not setting up a header will change how the body is handled. If the current code works for you, it should be fine without the header too I guess 🤷

Reply:
Wrapping a data structure into a `Json<>` tend to pass them to Serde for json serialization...